src/index.html

```diff
@@ -5,9 +5,61 @@
     <meta name="viewport"
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <title>Product cards</title>
-    <link rel="stylesheet" href="./style.css">
+    <link rel="stylesheet" href="./styles/index.scss">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin = "anonymous">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Product cards</h1>
+    <article class="card card--width" data-qa="card">
+      <div class="card__image-container">
+        <img src="./images/imac.jpeg"
+          alt="APPLE A1419 iMac 27&quot;
+          Retina 5K Monoblock (MNED2UA/A)"
+          class="card__image"
+        >
+      </div>
+
+      <h2 class="card__name">
+        APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+      </h2>
+
+      <p class="card__code">
+        Product code: 195434
+      </p>
+
+      <div class="stars card__stars">
+        <div class="stars__display stars__display--4">
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+        </div>
+
+        <p class="stars__review">
+          Reviews: 5
+        </p>
+      </div>
+
+      <div class="card__price">
+        <p class="card__label">
+          Price:
+        </p>
+
+        <p class="card__amount">
+          $2,199
+        </p>
+      </div>
+
+      <a href="#buy"
+        class="card__buy-button"
+        data-qa="hover">
+        Buy
+      </a>
```

Comment:
🥺Please, don't make your mentor cry. Formatting!
```
<a
  href="#buy"
  class="card__buy-button"
  data-qa="hover"
>
  Buy
</a>
```

Reply:
not fixed, @dsfreedom check please